packages/components/src/components/carousel/carousel-utils.ts

```diff
@@ -189,3 +191,18 @@ export const updatePagination = (paginationEl: HTMLElement, amountOfPages: numbe
     }
   }
 };
+
+export const getLangDirection = (el: HTMLElement): CarouselLanguageDirection => {
+  const dirAttribute = el.closest('[dir]')?.getAttribute('dir');
+  if (dirAttribute) {
+    return dirAttribute as CarouselLanguageDirection;
+  }
+
+  const lang = document.documentElement.lang;
+  if (lang && lang !== 'unknown') {
+    // @ts-expect-error "textInfo" is not supported in Firefox and not part of the types.
+    return new Intl.Locale(lang).textInfo?.direction || 'ltr';
```

Comment:
Not really standard yet
https://caniuse.com/mdn-javascript_builtins_intl_locale_gettextinfo

Reply:
Yes, as mentioned in the comment. But imho it degrades gracefully (and fallbacks to "ltr"). Why not offering support for Chrome/Edge/Safari compared to not offering it at all?